.github/workflows/benchmark.yml

```diff
@@ -202,16 +200,28 @@ jobs:
         run: |
           helm repo add zeebe-benchmark https://zeebe-io.github.io/benchmark-helm
           helm repo update
+          
+      - name: Find previous Helm chart release version
+        id: find-chart-release
+        run: |
+          version=$(helm list --namespace ${{ inputs.name }} --output json | jq --raw-output '.[0].chart' | cut --delimiter '-' --fields 3)
```

Comment:
```suggestion
          version=$(helm get metadata -o json --namespace ${{ inputs.name }} ${{ inputs.name }} | jq .version)
```

Reply:
If you looking for something to debug feel free I have no energy anymore today 🙈

Reply:
I think I made progress? https://github.com/camunda/zeebe/actions/runs/7573709800/job/20626562909#step:7:14